src/domain/products/components/prices-form/index.tsx

```diff
@@ -63,6 +64,16 @@ const PricesForm = ({ form }: Props) => {
           region_id: reg.id,
           amount: null,
           currency_code: reg.currency_code,
+          includes_tax: reg.includes_tax,
+        })
+      }
+
+      const index = fields.findIndex((field) => field.region_id === reg.id)
+
+      if (index !== -1 && fields[index].includes_tax !== reg.includes_tax) {
+        update(index, {
+          ...fields[index],
+          includes_tax: reg.includes_tax,
         })
```

Comment:
suggestion(non-blocking): Change naming both of these checks (region setting and currency setting) to avoid ambiguity

Reply:
Could be something like `const regionTaxSetting` and `const currencyTaxSetting`